### ISSUE TITLE
[Loss function] Implement pdf weighted bce loss

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#99]
Implement custom weighted binary cross entropy loss.

The weights should be based on pdf of the training dataset. The weights should be computed for buckets for efficiency.

Steps:
1. Identify thresholds for discretization (use equi-depth histograms with `n` buckets)
2. Precompute weights for the buckets. How?
3. Implement bce loss with weighting based on buckets

**Comment by kapiblue** [comment]
The proof of concept could use just two buckets originating from thresholding with some low epsilon value.
